tutorials/tree/dictionary/readTree.cpp

```diff
@@ -0,0 +1,48 @@
+// Author: Alvaro Tolosa-Delgado CERN 07/2023
```

Comment:
I believe that `readTree.cpp` should be renamed to `readTree.cxx` for symmetry with practically every other part of ROOT, similarly for `.hpp` to `.hxx`